src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts

```diff
@@ -0,0 +1,14 @@
+﻿namespace Windows.UI.ViewManagement {
+
+	export class ApplicationView {
+
+		public static setFullScreenMode(turnOn: boolean) {
+			if (turnOn) {
+				document.documentElement.requestFullscreen();
```

Comment:
You should use https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenEnabled to determine if it's available before opening it (and return `false` to `TryEnterFullScreenMode()`

Reply:
Done :-)